src/pages/studyView/StudyViewPage.tsx

```diff
@@ -750,6 +752,58 @@ export default class StudyViewPage extends React.Component<
                                                 </DefaultTooltip>
                                             </>
                                         )}
+                                        {(window as any).frontendConfig
```

Comment:
read config from SiteConfi

Reply:
I do not find the class SiteConfig anywhere.

Reply:
I have replaced all access to serverConfig via AppConfig.

Reply:
nudge